src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Build.ps1

```diff
@@ -827,12 +827,13 @@ class Build: ADOSVTBase
 
     hidden [ControlResult] CheckSettableAtQueueTimeForURL([ControlResult] $controlResult)
     {
-        $controlResult.VerificationResult = [VerificationResult]::Verify
+        $controlResult.VerificationResult = [VerificationResult]::Failed
```

Comment:
Not sure if we should change the result to failed. Similarly for release as well.

Reply:
Made changes in PartialManager.ps1 to capture the backup of those who are verify and failed , now onwards verify will also work.